docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -20,12 +20,37 @@ For this purpose, *Fast DDS Statistics module* exposes an extended DDS |Statisti
 Enable statistics DataWriters
 -----------------------------
 
-Statistics DataWriters are enabled using the method |enable_statistics_datawriter|.
-This method requires as parameters:
+Statistics DataWriters can be enabled in different ways.
+It can be done automatically (see :ref:`auto_enabling_statistics_datawriters`).
+Alternatively, Statistics DataWriters can be enabled at run time using one of two methods:
+|enable_statistics_datawriter| or |enable_statistics_datawriter_with_profile|.
+
+|enable_statistics_datawriter| method requires as parameters:
 
 * Name of the statistics topic to be enabled (see :ref:`statistics_topic_names` for the statistics topic list).
 * DataWriter QoS profile (see :ref:`statistics_datawriter_qos` for the recommended profile).
 
+It is possible to define specific desired QoS trough DataWriter profile on the `FASTRTPS_DEFAULT_PROFILES_FILE`
+(see :ref:`xml_profiles`).
+|enable_statistics_datawriter_with_profile| method enables a DataWriter by searching a specific DataWriter XML profile,
+or a generic statistics DataWriter profile.
+On those profiles, specific QoS can be set.
+For the creation of a Datawriter, the priority for setting its QoS is the following:
+
+* First, if a specific profile exists for the statistics topic, that one is applied.
+* If that is not the case but a generic profile for statistics DataWriters exists, that one is applied.
+* If no profile is defined in XML file, the recommended statistics QoS are applied.
+
+.. note::
+
+    The generic DataWriter profile defined in the `FASTRTPS_DEFAULT_PROFILES_FILE` XML needs to be named as
+    `GENERIC_STATISTICS_PROFILE`.
+
+|enable_statistics_datawriter_with_profile| method requires as parameters:
```

Comment:
I think this can be explained before explaining the QoS priority.

Reply:
Done: e1b04bb6 Refs #15014: Reubication of generic profile explanation to automatically enabling section.